pkg/apis/cyclone/v1alpha1/stage_template.go

```diff
@@ -79,5 +79,12 @@ type StageTemplateSpec struct {
 	// Pod kind workload
 	Pod PodWorkload `json:"pod,omitempty"`
 	// CRD kind workload
-	CRD CRDWorkload `json:"crd:omitempty"`
+	CRD CRDWorkload `json:"crd,omitempty"`
```

Comment:
```go
Arguments []Argument `json:"arguments,omitempty"`
```

Argument & ArgumentItem is a bit duplicated, can we merge them?  This makes the overall structures easier to reason about.

Reply:
`Argument ` defines argument, with `default` value, `description`. And `ArgumentItem` specify value for a argument, `ArgumentValue` may be a better choice.